scripts/completion/dvc.bash

```diff
@@ -8,35 +8,44 @@
 #   - https://opensource.com/article/18/3/creating-bash-completion-script
 #----------------------------------------------------------
 
-_dvc_commands='init destroy add import checkout run pull push fetch \
-              status repro remove move unprotect gc config remote metrics \
-              install root lock unlock pipeline commit'
+_dvc_commands='init get get_url destroy add remove move unprotect run \
+              repro pull push fetch status gc import import_url config \
```

Comment:
Should be import-url here though. I i recall correctly, `import-url` will be resolved to `_dvc_import_url`.
```suggestion
              repro pull push fetch status gc import import-url config \
```

Reply:
that's true, @efiop , but we would need to expand the options correctly :sweat_smile: 

I'll make a suggestion :)

Reply:
mhm, looks like I can't because the code wasn't edited, but @Naba7 , if you already applied @efiop 's suggestion, then you'll need to make the following change:

```diff
-     local options_list="_dvc_${COMP_WORDS[1]}"
+    local options_list="_dvc_${COMP_WORDS[1]//-/_}"
```

This will use replace the dash (`-`) with underscore (`_`) so it would call the correct function :)

Reply:
If we want to implement completion for sub-commands (eg. `metrics show`, `metrics add`), it will need to change `options_list` depending on `COMP_WORDS[2]`, some pseudo code could be:

```
command = COMP_WORDS[1]
sub_command = COMP_WORDS[2]
if command is "metrics" and sub_command is "show", then load `_dvc_metrics_show
```
maybe there's a way to generalize this (maybe something like `"_".join(COMP_WORDS)`)

Reply:
@Naba7 , take into account the following if you want to enable sub commands for https://github.com/iterative/dvc/issues/2069  ^^
Please, ping me if you have any doubt.